docs/source/intro/glossary.rst

```diff
@@ -0,0 +1,149 @@
+Glossary
+--------
+
+ASReview LAB makes use of standardized terminology for all communication
+regarding ASReview LAB and its underlying technology.
+
+.. glossary::
+
+  Active learning model
+    Active learning model is used to indicate how to select the next
+    :term:`record` to be reviewed by the :term:`user`. The model consists of
+    several elements: a feature extraction technique, a classifier, a balance
+    and a query strategy.
+
+  ASReview
+    ASReview stands for *Active learning for Systematic Reviews* or
+    *AI-assisted Systematic Reviews*, depending on context. Avoid this
+    explanation, only use as tagline.
+
+  ASReview CLI
+    ASReview CLI is the command line interface that is developed for advanced
+    options or for running simulation studies.
+
+  ASReview LAB
+    ASReview LAB is the graphical user interface.
+
+  ASReview project
+    ASReview project is an encompassing term for all work that is done by the
+    ASReview team members and ASReview :term:`contributors<contributor>`.
+
+  Contributor
+    Contributor is anyone who contributes to the :term:`ASReview project`
+    through GitHub.
+
+  Data
+    Data includes :term:`dataset`, prior knowledge, labels, and
+    :term:`notes<note>`.
+
+  Dataset
+    Dataset is the collection of :term:`records<record>` that the :term:`user`
+    :term:`imports<import>` and :term:`exports<export>`.
+
+  ELAS
+    ELAS stands for "Electronic Learning Assistant". It is the name of
+    :term:`ASReview` mascot. It is used for storytelling and to increase
+    explainability.
+
+  Export
+    Export is the action of exporting a :term:`dataset` or a :term:`project`
+    from :term:`ASReview LAB`.
+
+  Extension
+    Extension is the additional element to the :term:`ASReview LAB`, such as
+    the ASReview visualisation extension, or the ASReview CORD-19 extension.
```

Comment:
Do we want a link to the documentation about extensions?

Reply:
Added a link to the visualization. Maybe not for CORD-19 as it is deprecated.